engine/src/main/java/org/entando/entando/aps/system/services/page/model/PageDto.java

```diff
@@ -88,6 +91,11 @@ public PageDto(IPage page) {
                 .filter(widget -> widget != null)
                 .collect(Collectors.toList())
                 .size()));
+        if (null != page.getMetadata().getUpdatedAt()) {
+            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Here you can use the constants SystemConstants.API_DATE_FORMAT

Reply:
ok!